dockershrink/cli.py

```diff
@@ -52,111 +111,69 @@ def main():
         default=".dockerignore",
         help="Path to .dockerignore (default: ./.dockerignore)",
     )
-    optimize_parser.add_argument(
-        "--package-json",
-        type=str,
-        default=None,
-        help="Path to package.json (default: ./package.json or ./src/package.json)",
-    )
-    optimize_parser.add_argument(
-        "--output-dir",
-        type=str,
-        default="dockershrink.optimized",
-        help="Directory to save optimized files (default: ./dockershrink.optimized)",
-    )
-    optimize_parser.add_argument(
-        "--openai-api-key",
-        type=str,
-        default=None,
-        help="Your OpenAI API key to enable Generative AI features (alternatively, set the OPENAI_API_KEY environment variable)",
-    )
-    optimize_parser.add_argument(
-        "-v",
-        "--verbose",
-        action="store_true",
-        help="Print complete stack trace in case of failures",
-    )
+    add_common_arguments(optimize_parser)
     optimize_parser.set_defaults(func=optimize_command)
 
-    # Parse the arguments
-    args = parser.parse_args()
+    # Generate command
+    generate_parser = subparsers.add_parser(
+        "generate",
+        help="Generate new Dockerfile for NodeJS project",
+        description="""
+        Analyzes your NodeJS project and generates:
```

Comment:
I think the description should be written without any whitespace to its left (unless you intentionally want to see some space when the description is printed on the terminal)